tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -232,9 +232,27 @@ def build_functions_package(self, queue_name=None, target_subscription_ids=None)
 
 class AzureModeCommon:
     """ Utility methods shared across a variety of modes """
+
+    @staticmethod
+    def extract_resource_id(policy, event):
```

Comment:
nit: In the future someone will come along and be pretty confused why this method exists, might be nice to add a comment

```
"""
Searches for a resource id in the events resource id that will match the policy resource type. 
"""
```